app/src/main/java/com/compscieddy/meetinthemiddle/MapsActivity.java

```diff
@@ -209,4 +223,26 @@ public void onConnectionSuspended(int i) {
   public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
 
   }
+
+  public Bitmap croppedIcon(Bitmap bitmap) {
+    Bitmap output = Bitmap.createBitmap(bitmap.getWidth(),
+            bitmap.getHeight(), Bitmap.Config.ARGB_8888);
+    Canvas canvas = new Canvas(output);
+
+    final int color = 0xff000000;
```

Comment:
let's actually just use the color I already have: R.color.transparent - sorry forgot to mention this one before; forgot